src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -205,7 +205,7 @@ private CompletableFuture<String> placeOrder(CompositePurchaseOrder compPO) {
       return restClient.post(ORDERS_ENDPOINT, JsonObject.mapFrom(compPO))
         .map(body -> {
           logger.info("Response from mod-orders: \n {}", body::encodePrettily);
```

Comment:
lets also delete response body from log, probably we missed this place when deleting similar logs from code, logs should not contain sensitive info

Reply:
It will make harder to debug issues, propose to remain this log but only for onFailure section, in this case we will not expose order content and will have more info about errors

Reply:
@SerhiiNosko why not change it to `logger.debug` ? I think that message is not necessary when not in debug mode.